src/Cooked/MockChain/GenerateTx.hs

```diff
@@ -131,15 +121,16 @@ txSkelToBodyContent TxSkel {..} = do
   return Cardano.TxBodyContent {..}
 
 generateBodyContent ::
```

Comment:
Everything is clear from the parameters names but we could add brief haddock descriptions for the function and parameters

Reply:
Same comment as before: I believe we could do a pass of haddock comments for this module, but maybe in a nother PR.

Reply:
Agreed :+1: